### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by BalazsR**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.vanniktech.code.quality.tools'
 
 buildscript {
     ext.versions = [
-            kotlin: '1.3.41',
-            minSdk: 16,
-            compileSdk: 28,
-            androidTools: '26.5.0',
+            kotlin: '1.3.72',
```
Comment (WonderCsabo) [inline build.gradle]: We have later versions than this. Is this intentional for backward compatibility reasons?

File: build.gradle [hunk 2]
```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.vanniktech.code.quality.tools'
...
+            minSdk: 21,
+            compileSdk: 30,
+            androidTools: '27.1.3',
```
Comment (WonderCsabo) [inline build.gradle]: Same question here.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```
Comment (WonderCsabo) [inline gradle/wrapper/gradle-wrapper.properties]: Same here

**Review by WonderCsabo — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (BalazsR) [reply build.gradle]: updated

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (BalazsR) [reply gradle/wrapper/gradle-wrapper.properties]: updated

**Pull request merged**